exonum/src/sandbox/sandbox.rs

```diff
@@ -642,6 +670,16 @@ impl Drop for Sandbox {
     }
 }
 
+impl ConnectList {
+    // Creates from state::peers, needed only for testing.
+    #[doc(hidden)]
+    pub fn from_peers_for_testing(peers: &HashMap<PublicKey, Connect>) -> Self {
```

Comment:
Since it's a sandbox, I guess you can skip the `for testing` part since it's obvious.